docs-next/pages/whats-new.mdx

```diff
@@ -0,0 +1,65 @@
+import { Markdown } from '../components/Page';
+
+# What's New in Keystone Next
+
+> Note: this page is WIP, we have a lot more things to add to it!
+
+In this major update, we've focused on improving Keystone's **interfaces**, including the way you
+configure and run Keystone projects, and a whole new Admin UI.
+
+We're also adding powerful new features to make Keystone the best headless content management system
+around, especially when you're using a component-based front end like React and Vue.
```

Comment:
```suggestion
around, especially when you're using a component-based front-end like React and Vue.
```
I'm not too fussed on frontend vs front end vs front-end. This brings it in line with another instance of "front-end" below.

Reply:
Yeah good catch, thanks. I caught a few myself and was thinking we really need to do a pass for consistency when we're done.